modules/veteran_verification/spec/requests/api_docs_request_spec.rb

```diff
@@ -18,4 +18,12 @@
       JSON.parse(response.body)
     end
   end
+
+  describe '#get /docs/v0/status' do
+    it 'should return Open API Spec v3 JSON' do
+      get '/services/veteran_verification/docs/v0/status'
+      expect(response).to have_http_status(:ok)
+      JSON.parse(response.body)
```

Comment:
Is this just here so that if `JSON.parse` errors the test will fail...rather than expecting something?

Reply:
yeah. just making sure we're returning valid JSON.